saos-import/src/main/java/pl/edu/icm/saos/importer/commoncourt/judgment/remove/CcjRemoverReader.java

```diff
@@ -0,0 +1,73 @@
+package pl.edu.icm.saos.importer.commoncourt.judgment.remove;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+import org.springframework.batch.item.ExecutionContext;
+import org.springframework.batch.item.ItemStreamException;
+import org.springframework.batch.item.ItemStreamReader;
+import org.springframework.batch.item.NonTransientResourceException;
+import org.springframework.batch.item.ParseException;
+import org.springframework.batch.item.UnexpectedInputException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import pl.edu.icm.saos.persistence.model.Judgment;
+import pl.edu.icm.saos.persistence.model.SourceCode;
+import pl.edu.icm.saos.persistence.repository.JudgmentRepository;
+
+/**
+ * Spring batch reader of common court judgments
+ * that should be removed
```

Comment:
... that have been removed at the source side... (?)
The reader does not know what the processor will do with them.